webapp/src/main/java/com/box/l10n/mojito/entity/Repository.java

```diff
@@ -58,7 +59,7 @@
     private DropExporterType dropExporterType;
 
     @JsonView(View.RepositorySummary.class)
-    @JsonManagedReference
+    @JsonManagedReference("repositoryLocales")
```

Comment:
Just curious, why do you have a string after this annotation, but not some others, like the one in line 76 below?

Reply:
Is it just so it will match the `@JsonBackReference` in `RepositoryLocale.java`?